src/components/UI/Textarea/Textarea.tsx

```diff
@@ -1,8 +1,11 @@
 import styled from "@emotion/styled";
-import { StyledInput } from "../../../common/styles/Input";
+import tw from "twin.macro";
 
 export const TextareaDefault = styled.textarea`
-  ${StyledInput()}
+  ${tw`border border-solid border-gray-300 bg-white py-1 px-2 mb-2`}
+  &::placeholder {
+    ${tw`italic text-gray-500 text-base`}
+  }
   min-height: 140px;
   width: 100%;
   resize: none;
```

Comment:
maybe let's delete this and its storybook totally and use in `EmailForm` directly (it's only used there)

```
const TextArea = styled.textarea`
  min-height: 140px;
  &::placeholder {
    ${tw`italic text-gray-500 text-base`}
  }
`;

<TextArea className="border border-solid border-gray-300 bg-white py-1 px-2 mb-2 w-full resize-none" />
```

i not sure if u can do `@apply <tailwindclass>`, if can best. if cannot then leave it to tw macro for the placeholder styles i guess

Reply:
yup ok, i'll do the suggested change, not too sure about `@apply <tailwindclass>`, is that tailwind's way of mix-ins?

Reply:
um u can use it in `tailwind.css` that file there, but internally i not sure if they will magic it not